src/Button/Button.Styled.js

```diff
@@ -0,0 +1,173 @@
+import styled from 'styled-components';
+
+
+export const StyledButton = styled.button`
+    color: #F6F9FC;
+    font:14px 'Arial';
+    border: none;
+    background-color:#FF4E54;
+    padding:11px 28px;
+    margin:10px;
+    cursor: pointer;
+`;
+export const SecondButton = styled.button`
```

Comment:
@KBhautik6 use prefix `Styled` for this

Reply:
Basically it's function given by styled - component so , I use this thing